resources/extruders/winbo_dragonl4_extruder.def.json

```diff
@@ -0,0 +1,19 @@
+{
+    "id": "winbo_dragonl4_extruder",
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "winbo_dragonl4",
+        "position": "0"
+    },
+
+    "overrides": {
+        "extruder_nr": {
+            "default_value": 0,
+            "maximum_value": "2"
+        },
+        "machine_nozzle_offset_x": { "default_value": 0.0 },
+        "machine_nozzle_offset_y": { "default_value": 0.0 }
+    }
```

Comment:
Need to define here the `material_diameter` and `machine_nozzle_size`, instead of in the machine definition.